lib/events.js

```diff
@@ -1,5 +1,7 @@
 /*!
- * Copyright (c) 2012-2022 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2012-2024 Digital Bazaar, Inc.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Bummer that we need this in every file.

Reply:
There is an alternative for spelling out copyright in a `.reuse/dep5` file--which I've added for formats where comments are cumbersome/odd (Markdown uses HTML comments) or not available (`.json`): https://github.com/digitalbazaar/bedrock/pull/112/commits/45a2af2649ea2c46d73b309763aea52486a343c2

The more explicit we are, the less confusion the could/would be when we include any outside contribution, file, or directory (which would contain it's own copyright info).

The wildcard approach of a single `LICENSE` file just isn't as clear.